src/components/playground/playground.scss

```
@@ -189,6 +189,11 @@ section.example {
 }
 
 .show-case_component {
+    font-family: var(--kompendium-example-font-family);
+    font-size: var(--kompendium-example-font-size);
+    line-height: var(--kompendium-example-line-height);
+    color: var(--kompendium-example-color);
```

It makes no practical difference, at least not in the version of Chrome I'm using, but I don't like that these default to what I assume is essentially an invalid declaration, since the css vars are not set by default.

```suggestion
    font-family: var(--kompendium-example-font-family, inherit);
    font-size: var(--kompendium-example-font-size, inherit);
    line-height: var(--kompendium-example-line-height, inherit);
    color: var(--kompendium-example-color, inherit);
```